app/controllers/testonly/BondsMongoController.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.testonly
+
+import config.Refined.Max5000
+import controllers.actions.IdentifyAndRequireData
+import eu.timepit.refined._
+import models.{Money, SchemeHoldBond}
+import models.SchemeId.Srn
+import pages.nonsipp.unregulatedorconnectedbonds._
+import play.api.mvc.MessagesControllerComponents
+import services.SaveService
+import shapeless._
+
+import java.time.LocalDate
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+
+class BondsMongoController @Inject()(
```

Comment:
I thought this is in @westwater 's  branch @narendrapaduchuri ?

Reply:
Yes @kberec , George also added this end point.. I will wait until George ticket merge to main